From 1a90fdc44414fc4acc522b7fbab77988c22400f0 Mon Sep 17 00:00:00 2001
From: Jammy Zhou <Jammy.Zhou@amd.com>
Date: Sat, 6 Jun 2015 05:02:23 +0800
Subject: [PATCH 38/67] amdgpu: cleanup gds specific alloc/free functions

amdgpu_bo_alloc/free can be used with GDS/OA/QWS domains specified

Signed-off-by: Jammy Zhou <Jammy.Zhou@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Arindam Nath <arindam.nath@amd.com>
---
 amdgpu/amdgpu.h | 60 ---------------------------------------------------------
 1 file changed, 60 deletions(-)

diff --git a/amdgpu/amdgpu.h b/amdgpu/amdgpu.h
index 44b9d17..c4c0cd4 100644
--- a/amdgpu/amdgpu.h
+++ b/amdgpu/amdgpu.h
@@ -251,28 +251,6 @@ struct amdgpu_gds_resource_info {
 	uint32_t   oa_per_compute_partition;
 };
 
-
-
-/**
- *  Structure describing result of request to allocate GDS
- *
- *  \sa amdgpu_gpu_resource_gds_alloc
- *
-*/
-struct amdgpu_gds_alloc_info {
-	/** Handle assigned to gds allocation */
-	amdgpu_bo_handle resource_handle;
-
-	/** How much was really allocated */
-	uint32_t   gds_memory_size;
-
-	/** Number of GWS resources allocated */
-	uint32_t   gws;
-
-	/** Number of OA resources allocated */
-	uint32_t   oa;
-};
-
 /**
  * Structure describing IB
  *
@@ -781,44 +759,6 @@ int amdgpu_query_gds_info(amdgpu_device_handle dev,
 			struct amdgpu_gds_resource_info *gds_info);
 
 
-/**
- * Allocate GDS partitions
- *
- * \param   dev	       - \c [in] Device handle. See #amdgpu_device_initialize()
- * \param   gds_size   - \c [in] Size of gds allocation. Must be aligned
- *				accordingly.
- * \param   alloc_info - \c [out] Pointer to structure to receive information
- *				  about allocation
- *
- * \return   0 on success\n
- *          >0 - AMD specific error code\n
- *          <0 - Negative POSIX Error code
- *
- *
-*/
-int amdgpu_gpu_resource_gds_alloc(amdgpu_device_handle dev,
-				   uint32_t gds_size,
-				   struct amdgpu_gds_alloc_info *alloc_info);
-
-
-
-
-/**
- * Release GDS resource. When GDS and associated resources not needed any
- * more UMD should free them
- *
- * \param   dev	   - \c [in] Device handle. See #amdgpu_device_initialize()
- * \param   handle - \c [in] Handle assigned to GDS allocation
- *
- * \return   0 on success\n
- *          >0 - AMD specific error code\n
- *          <0 - Negative POSIX Error code
- *
-*/
-int amdgpu_gpu_resource_gds_free(amdgpu_bo_handle handle);
-
-
-
 /*
  * GPU Execution context
  *
-- 
1.9.1

